build.gradle

```
@@ -14,127 +14,83 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 buildscript {
   ext {
+    korkVersion = "5.2.6"
+    fiatVersion = "1.0.0"
```

we should at least bump to `fiat 1.0.3` - otherwise we are regressing with what's there now, ideally, just pick up latest 
```suggestion
    fiatVersion = "1.0.4"
```